README.md

```diff
@@ -1,5 +1,9 @@
 # Central repository for work on libp2p
 
+<a href="http://libp2p.io/"><img src="https://img.shields.io/badge/project-libp2p-yellow.svg?style=flat-square" /></a>
+<a href="http://webchat.freenode.net/?channels=%23libp2p"><img src="https://img.shields.io/badge/freenode-%23libp2p-yellow.svg?style=flat-square" /></a>
+<a href="https://waffle.io/libp2p/libp2p"><img src="https://img.shields.io/badge/pm-waffle-yellow.svg?style=flat-square" /></a>
+
 [![dependency status](https://deps.rs/repo/github/libp2p/rust-libp2p/status.svg)](https://deps.rs/repo/github/libp2p/rust-libp2p)
```

Comment:
Maybe we can put the dependency status badge on the same line as the other three?